content/docs/start/index.md

```diff
@@ -50,10 +50,10 @@ $ git commit -m "Initialize DVC"
 
 Now you're ready to DVC!
 
-The value of DVC's many feature sets is best understood from different angles.
-Pick a trail below to see an overview of all features from that perspective:
+The value of DVC's several feature sets is best understood from different
+angles. Pick from the trails below to learn about DVC from that perspective:
 
-### Data Management
+### Data Management Trail
 
 - **[Data and model versioning]** is the base layer of DVC for large files,
```

Comment:
Another option would be to add a link to the top of the trail only and avoid linking to each sub-page here.